### ISSUE TITLE
Add comment-multi-line-indentation

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#4781]
> What is the problem you're trying to solve?

I want to enforce consistent indentation in my comments.

For example, flush:

```css
/*
Default Styles and layout for Conf-pages
Do not custom this CSS
*/
```

Or doc blocks:

```css
/* Document
 * ========================================================================== */

/**
 * Add border box sizing in all browsers (opinionated).
 */

```

See: https://github.com/stylelint/stylelint/issues/4776#issuecomment-629768045

> What solution would you like to see?

A new rule that enforces this. Based on the [`indentation` rule](https://stylelint.io/user-guide/rules/indentation), it'd have an `indentClosingMarker` option similar to [`indentClosingBrace` one in `indentation`](https://stylelint.io/user-guide/rules/indentation#indentclosingbrace-truefalse).

Examples:

```json
{ "comment-multi-line-indentation": 0 }
```

Would allow:

```css
/*
Default Styles and layout for Conf-pages
Do not custom this CSS
*/
```

And disallow:

```css
/*
     Default Styles and layout for Conf-pages
 Do not custom this CSS
     */
```

```json
{ "comment-multi-line-indentation": [1, { "indentClosingMarker": true }] }
```

Would allow:

```css
/**
 * Add border box sizing in all browsers (opinionated).
 */
```

And disallow:

```css
/**
       * Add border box sizing in all browsers (opinionated).
*/
```

**Comment by jeddy3** [comment]
As they are no objections to this rule, I've labelled the issue as ready to implement. @ThiefMaster Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to add a new rule](https://stylelint.io/developer-guide/rules#add-a-rule) in the Developer guide.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to create the rule [as a plugin](https://stylelint.io/developer-guide/plugins).
